_pages/en/product-fw-guidance.md

```diff
@@ -0,0 +1,352 @@
+        />
+  </a>
+  <figcaption>Figure 3. Three types of change and their funding coverage</figcaption>
+</figure>
+
+Funding RUN activities continue to be used for “keeping the lights on”.
+
+Projects are used to coordinate large change, such as changing the enterprise through a transformation or needing to coordinate change over multiple products.  
+
+In between however, there is an opportunity to allow rapid, small, changes to software: enabling continuous improvements. Typically on existing business processes, to enhance user experience, and continuously address cyber security risks.
+
+A few things are necessary to enable such continuous improvements, which is the focus on this Directive and Standard.
+
+<figure>
+<a href="{{ site.baseurl }}/assets/images/product-roadmap-roles.png">
+        <img class="center" src="{{ site.baseurl }}/assets/images/product-roadmap-roles.png"
+        alt="The Image has roughly three sections. The section on the left contains three icons of documents each labeled Projects, Improvements, and Maintenance. An arrow goes from them to the second section and is labeled Feed Requirements. The section section is labeled Job Bank, representing the Product Job Bank with its Backlog (which looks like a repository) and a time arrow going right, with sections saying 3 months, 6 months, and 12 months. At the end, the arrow forks into two possible future A and B. The third section is under the Job Bank section. It contains three icons: a Product Manager, a Product Owner, and a Product Team. The Product Manager icon has the text Strategic Role, Produces the Product Roadmap, Performs Corporate Planning (e.g. secure funding). The Product Owner icon has the text Tactical Role, Uses the Product Backlog to scope releases, Has authority to make decisions. The Product Team icon has the text Cross-functionally skilled, Cross-Branch membership, Both development and operations, May be 100% funded by the Program Official."
+        />
+  </a>
+  <figcaption>Figure 4. Projects bringing work to Product Teams</figcaption>
+</figure>
+
+The different kinds of work affecting a particular product are requirements feeding that product's backlog.
+Product Teams perform their own improvements and maintenance activities, while external Projects _bring_ work to them.
+
+To manage and coordinate all this demand, the **Product Manager** is used. As a strategic role she looks beyond a 6 month time horizon. The Product Manager uses the product's roadmap as both a communication tool AND a negotiating one.

```

Comment:
We are keeping the gender binary? :-(

Reply:
> We are keeping the gender binary? :-(

I tried the "they" and felt it reads bizarre. Maybe because I'm not used to this change in language yet. Unless I put it plural (e.g. "Product Managers are used.... they look beyond a 6 month time horizon").

My idea here was that, traditionally, masculine is used for generic wordings. I wanted to use both a female representation (product manager) and a male one (product owner) to step outside the generic use of masculine.